Sources/WalletConnect/Engine/SessionEngine.swift

```diff
@@ -9,11 +9,10 @@ final class SessionEngine {
     var onSessionResponse: ((Response)->())?
     var onSessionSettle: ((Session)->())?
     var onSessionRejected: ((String, SessionType.Reason)->())?
-    var onSessionUpdate: ((String, Set<Account>)->())?
-    var onSessionUpgrade: ((String, SessionPermissions)->())?
+    var onSessionUpdateAccounts: ((String, Set<Account>)->())?
     var onSessionExtended: ((Session) -> ())?
     var onSessionDelete: ((String, SessionType.Reason)->())?
-    var onNotificationReceived: ((String, Session.Notification)->())?
+    var onNotificationReceived: ((String, Session.Event)->())?
```

Comment:
I think we can change this var to `onEventReceived`, now that we're dropping the "notification" terminology